molSimplify/Classes/mol3D.py

```diff
@@ -5336,17 +5338,19 @@ def is_edge_compound(self, transition_metals_only=True):
                 if len(lig) >= 2 and not set(lig) in _el:
                     edge_ligands.append([set(lig)])
                     _el.append(set(lig))
-                    break
+                    # break
             num_edge_lig = len(edge_ligands)
             info_edge_lig = [
                 {"natoms_connected": len(x[0])} for x in edge_ligands]
+            edge_lig_atoms = [
+                {"atom_idxs": x[0]} for x in edge_ligands]
         else:
-            num_edge_lig, info_edge_lig = 0, list()
-        return num_edge_lig, info_edge_lig
+            num_edge_lig, info_edge_lig, edge_lig_atoms = 0, list(), list()
+        return num_edge_lig, info_edge_lig, edge_lig_atoms
 
     def get_geometry_type(self, dict_check=False, angle_ref=False, num_coord=False,
                           flag_catoms=False, catoms_arr=None, debug=False,
-                          skip=False, transition_metals_only=False):
+                          skip=False, transition_metals_only=False, num_recursions=[0, 0]):
```

Comment:
This part is the biggest problem for me, maybe you could explain it to me in person at some point? Why is the parameter called recursions? (Means something completely different to me). Most importantly how is this function supposed to be used? I would expect that it figures out that there are sandwich and edge ligands in there, not that I need to tell it beforehand...